### PR TITLE
Add Button Component

### PR DESCRIPTION
**Pull request by parush-uppal**
Added button component and props

### CODE REVIEW

File: components/Button.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
```
Comment (HassanBahati) [inline components/Button.js]: @parush-uppal could we please change the folder structure to `/components/Button/index.js` do that we nest the button component inside a folder called `Button`. 

This is to enable us write tests for that component later on.

Comment (HassanBahati) [reply components/Button.js]: @parush-uppal instead of writing the text prop as `t` could we change it to `text` so as to enable easy code readability.

Comment (HassanBahati) [reply components/Button.js]: could the following props names be changes 

- `t` to `text`
- `tc` to `textColor`
- `bc` to `backgroundColor`

this is to ensure easy code readability

**Review by HassanBahati — CHANGES_REQUESTED** [review]

File: components/Button.js (hunk 1 above)
Comment (parush-uppal) [reply components/Button.js:1]: Did the naming changes in the props just wanted to know about button component do you want me to create a separate button folder inside components or want me to move button folder inside pages

**Comment by parush-uppal** [comment]
Changed The Props Names and created a separate folder button inside components containing button component

**Review by HassanBahati — APPROVED** [review]

**Comment by HassanBahati** [comment]
Looks Good To Me

**Comment by HassanBahati** [comment]
@parush-uppal thank you for the contribution :tada:

**Pull request merged**